### ISSUE TITLE
Remove simplejson dependency #2308

### ISSUE THREAD

**Issue by cclauss** [internetarchive/infogami#133]
Like  internetarchive/openlibrary #2308 but for infogami.
* [ ] infogami/infobase/_dbstore/read.py
* [ ] infogami/infobase/_dbstore/save.py
* [ ] infogami/infobase/_dbstore/store.py
* [ ] infogami/infobase/_json.py
* [ ] infogami/infobase/client.py
* [ ] infogami/infobase/common.py
* [ ] infogami/infobase/core.py
* [ ] infogami/infobase/dbstore.py
* [ ] infogami/infobase/infobase.py
* [ ] infogami/infobase/logger.py
* [ ] infogami/infobase/logreader.py
* [ ] infogami/infobase/readquery.py
* [ ] infogami/infobase/tests/test_client.py
* [ ] infogami/infobase/tests/test_save.py
* [ ] infogami/infobase/writequery.py
* [ ] infogami/plugins/api/code.py
* [ ] infogami/utils/app.py
* [ ] infogami/utils/flash.py
* [ ] test/test_infobase.py
* [ ] tests/test_infogami/test_pages.py

**Comment by Enigma04** [comment]
can I take up this issue if its still a problem?

**Comment by cclauss** [comment]
Sure @Enigma04, Thanks!  Please approach ___across many pull requests___ in the following way...
1. PR # 1 only modifies the tests (four files) and ensures that tests are running and passing before and after.
2. PR # 2 utils (two files)
3. PR # 3 api/code (just one file)
4. Repeat the cycle with just one or two files in each PR (no big bang PRs with lots of files please)
5. Do readquery and writequery files in the second last PR
6. Do the _dbstore files in the last PR
Hopefully, @dherbst can help me review these PRs.

**Comment by Enigma04** [comment]
I have a doubt though, what should I replace in the programs that still uses simpleJSON in their code? Like I can remove the dependencies but it causes errors as we are using simplejson in the code. I am new to open source so if you could clear that for me it'll be great!

**Comment by cclauss** [comment]
Yes.  Please use the Python standard library’s json module as the replacement for simplejson.

**Comment by Enigma04** [comment]
I have made the pull request for the test files. Can you check?
